### PR TITLE
Remove broken flag from 20H2 images as now supported by ACR

### LINKED ISSUE #552: Issue to track 20H2 support
This issue will track when full windows 20H2 support will be back and we can publish those images to ACR. At the moment, we have provided images and corresponding dockerfiles for nanoservercore and windowsservercore 20H2 distributions but can't publish it to ACR. 

An issue has been created on the ACR repo too: https://github.com/Azure/acr/issues/532

## Check list

- [ ] Verified that this is not a [Known Issue](https://github.com/PowerShell/PowerShell-Docker/wiki/Known-Issues)
- [ ] Verified this is not an issues in the underlying windows container that should be reported to [Windows Container User Voice](https://windowsserver.uservoice.com/forums/304624-containers)

### PR DESCRIPTION
**Pull request by vboulineau**
## PR Summary

Remove broken flag from 20H2 images as now supported by ACR. Fixes #552.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [x] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [x] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/lts/windowsservercore20H2/meta.json [hunk 1]
```diff
@@ -16,5 +16,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```
Comment (anamnavi) [inline release/lts/windowsservercore20H2/meta.json:19]:
```suggestion
```

**Review by RDIL — APPROVED** [review]
LGTM.

File: release/lts/windowsservercore20H2/meta.json (hunk 1 above)
Comment (anamnavi) [inline release/lts/windowsservercore20H2/meta.json:19]:
```suggestion
```

File: release/preview/nanoserver20H2/meta.json [hunk 2]
```diff
@@ -17,5 +17,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```
Comment (anamnavi) [inline release/preview/nanoserver20H2/meta.json:20]:
```suggestion
```

File: release/preview/windowsservercore20H2/meta.json [hunk 3]
```diff
@@ -17,5 +17,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```
Comment (anamnavi) [inline release/preview/windowsservercore20H2/meta.json:20]:
```suggestion
```

File: release/stable/nanoserver20H2/meta.json [hunk 4]
```diff
@@ -17,5 +17,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```
Comment (anamnavi) [inline release/stable/nanoserver20H2/meta.json:20]:
```suggestion
```

File: release/stable/windowsservercore20H2/meta.json [hunk 5]
```diff
@@ -17,5 +17,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```
Comment (anamnavi) [inline release/stable/windowsservercore20H2/meta.json:20]:
```suggestion
```

**Review by anamnavi — APPROVED** [review]
LGTM, however I'm going to remove the `isBroken` tag altogether

File: release/stable/windowsservercore20H2/meta.json (hunk 5 above)
Comment (RDIL) [reply release/stable/windowsservercore20H2/meta.json:20]: Oh yeah, that actually is a better idea.  How did I not think of that 😄

**Comment by TravisEz13** [comment]
Thank you for the PR and thanks @anamnavi  for the help.

**Pull request merged**